src/components/App.js

```diff
@@ -171,7 +171,10 @@ function MapComponent() {
           map,
           router,
         );
-        window.history.pushState('treetracker', '', path);
+        // changing the url in history state
+        // this allow website to load page back
+        const historyState = { ...window.history.state, url: path };
+        window.history.replaceState(historyState, '', path);
```

Comment:
I think this makes the app can not remember the previous jumps, for the case that doesn't includes bounds, like: /top, /wallets/xxxx, because we always replaceState, please try to jump on dev env is difficult, you set this to connect to prod env: https://github.com/Greenstand/treetracker-web-map-client#how-to-connect-to-production-data-locally

Reply:
> I think this makes the app can not remember the previous jumps, for the case that doesn't includes bounds, like: /top, /wallets/xxxx, because we always replaceState, please try to jump on dev env is difficult, you set this to connect to prod env: https://github.com/Greenstand/treetracker-web-map-client#how-to-connect-to-production-data-locally

Hi Dadiochen,
Thanks for taking a look at my code and giving instructions on how to connect to prod env, my local is working much better now. 
From what I understand, you said that the website doesn't remember the previous URL that doesn't include the bounds since I use the replaceState. So what is happening is the replaceState only happens when the bounds are changed which is a triggered event 
https://github.com/QuanGiap/treetracker-web-map-client/blob/Ignore-history/back-for-updating-the-bounds-on-url-address-%23725/src/components/App.js#L167
So when you to other pages, it still stores the page you have visited in the history.